specs/ops-matters-discussions.md

```diff
@@ -0,0 +1,56 @@
+# Use Github Issues on this repository (CCI-MOC/ops-docs) for ops discussions
+
+We need a place to have discussion about our operational policy.
+The idea is to create issues on github and discuss the policy there.
+
+## Problem Description
+
+It is not ideal to discuss operational matters in group chats because:
+
+* it's not easier to add more people to the discussion
+* messages get lost in long chat threads.
+* chats are not async enough.
+
+## Policy
+
+Users will create issues on this repository to discuss operational matters.
+People can then comment on the github issues, or respond via email to take
+part in those discussions.
```

Comment:
I think the discussion being stuck in a proprietary tool does concern me a bit in the long term. Yes, it is more user friendly than emails, but what is the audience tech level for these discussion and would the user friendliness of the tool make a difference in their participation? On the other hand, we're already having the code review discussion in PRs on GitHub, so my point is moot.